services/core/shared/database/knex_migrations/blocks.js

```diff
@@ -13,30 +13,14 @@
  * Removal or modification of this copyright notice is prohibited.
  *
  */
-const config = require('../../../config');
-
-const tableName = config.db.collections.blocks.name;
+const tableName = 'blocks';
 
 exports.up = knex => knex.schema
     .createTable(tableName, table => {
-        // Indexed properties
         table.string('id').primary();
         table.integer('height').notNullable().index();
         table.integer('unixTimestamp').notNullable().index();
```

Comment:
is there any need to index it as a `unixTimestamp` instead of `timestamp` when indexing blocks. According to the document, `Lisk Service SDK v5 Update` it should be indexed as timestamp. 
Just a thought, wouldn't see any issues using `unixTimestamp` as well.

Reply:
I continue to use `unixTimestamp` to stay in synchronisation with the existing implementations corresponding SDKv4 and older.
Also, `timestamp` in `Lisk Service SDK v5 Update` document relates to the request URL filtering params which are already supported at the Lisk Service Gateway.